modules/installation-gcp-permissions.adoc

```diff
@@ -29,6 +29,275 @@ account requires the following permissions. If you deploy your cluster into an e
+* `compute.instanceGroups.get`
+* `compute.instanceGroups.list`
+* `compute.instanceGroups.update`
+* `compute.instanceGroups.use`
+* `compute.instances.create`
+* `compute.instances.delete`
+* `compute.instances.get`
+* `compute.instances.list`
+* `compute.instances.setLabels`
+* `compute.instances.setMetadata`
+* `compute.instances.setServiceAccount`
+* `compute.instances.setTags`
+* `compute.instances.use`
+* `compute.machineTypes.get`
+* `compute.machineTypes.list`
+====
+
+.Required for creating storage resources
+[%collapsible]
+====
+* `storage.buckets.create`
+* `storage.buckets.delete`
+* `storage.buckets.get`
+* `storage.buckets.list`
+* `storage.objects.create`
+* `storage.objects.delete`
+* `storage.objects.get`
+* `storage.objects.list`
+====
+
+.Required permissions for creating health check resources
+[%collapsible]
+====
+* `compute.healthChecks.create`
+* `compute.healthChecks.get`
+* `compute.healthChecks.list`
+* `compute.healthChecks.useReadOnly`
+* `compute.httpHealthChecks.create`
+* `compute.httpHealthChecks.get`
+* `compute.httpHealthChecks.list`
+* `compute.httpHealthChecks.useReadOnly`
+====
+
+.Required permissions to get GCP zone and region related information
+[%collapsible]
+====
+* `compute.globalOperations.get`
+* `compute.regionOperations.get`
+* `compute.regions.list`
+* `compute.zoneOperations.get`
+* `compute.zones.get`
+* `compute.zones.list`
+====
+
+.Required permissions for checking services and quotas
+[%collapsible]
+====
+* `monitoring.timeSeries.list`
+* `serviceusage.quotas.get`
+* `serviceusage.services.list`
+====
+
+.Optional IAM permissions for installation
+[%collapsible]
+====
+* `iam.roles.get`
+====
+
+.Optional Images permissions for installation
+[%collapsible]
+====
+* `compute.images.list`
+====
+
+.Optional permission for running gather bootstrap
+[%collapsible]
+====
+* `compute.instances.getSerialPortOutput`
+====
+
+.Required permissions for deleting network resources
+[%collapsible]
+====
+* `compute.addresses.delete`
+* `compute.addresses.deleteInternal`
+* `compute.addresses.list`
+* `compute.firewalls.delete`
+* `compute.firewalls.list`
+* `compute.forwardingRules.delete`
+* `compute.forwardingRules.list`
+* `compute.networks.delete`
+* `compute.networks.list`
+* `compute.networks.updatePolicy`
+* `compute.routers.delete`
+* `compute.routers.list`
+* `compute.routes.list`
+* `compute.subnetworks.delete`
+* `compute.subnetworks.list`
+====
+
+.Required permissions for deleting load balancer resources
+[%collapsible]
+====
+* `compute.regionBackendServices.delete`
+* `compute.regionBackendServices.list`
+* `compute.targetPools.delete`
+* `compute.targetPools.list`
+====
+
+.Required permissions for deleting DNS resources
+[%collapsible]
+====
+* `dns.changes.create`
+* `dns.managedZones.delete`
+* `dns.managedZones.get`
+* `dns.managedZones.list`
+* `dns.resourceRecordSets.delete`
+* `dns.resourceRecordSets.list`
+ifdef::template[]
+* `dns.resourceRecordSets.update`
```

Comment:
This permission is required for creating (rather than deleting) DNS resources for user-provisioned GCP infrastructure.

Reply:
Nice catch. Thanks.

Reply:
Addressed